checkstyle-tester/diff.groovy

```diff
@@ -275,6 +287,11 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     summaryIndexHtml << ('<strong>WARNING: Excludes are ignored by diff.groovy.</strong>')
     summaryIndexHtml << ('</span></h3>')
     printReportInfoSection(summaryIndexHtml, checkstyleBaseReportInfo, checkstylePatchReportInfo, projectsStatistic)
+
+    def textTransform = getTextTransform();
+    def listOfProjectsFile = new File(listOfProjects)
+    def listOfProjectsHtml = generateConfigFileHtml(diffDir, listOfProjectsFile, textTransform, summaryIndexHtml)
```

Comment:
Method isn't returning anything now. `listOfProjectsHtml` can be removed.

Reply:
Of course. Fixed.